Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Payout.yaml

```
@@ -110,6 +110,30 @@
         userActionType: ApiAuth DRIVER_OFFER_BPP_MANAGEMENT DRIVERS PAYOUT_MANAGEMENT
 
+  - POST: # update vpa's
+      endpoint: payout/updateVPA
+      auth: ApiAuthV2
+      request:
+        type: UpdateVpaReq
+      response:
+        type: APISuccess
+      migrate:
+        endpointV2:
+      migrate:
```

YAMLlint (1.35.1)

[error] duplication of key "migrate" in mapping

(key-duplicates)